Workbooks/Intune Devices/Devices/IntuneDevices.workbook

```diff
@@ -0,0 +1,351 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Intune Devices Activity "
+      },
+      "name": "Intune Devices Activity Title"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "## Parameters"
+      },
+      "name": "Text - Parameters"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "e5c8c9bc-d4ae-4434-95d7-d6d2128399cf",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "type": 4,
+            "description": "Time range chosen to show the Number of Devices Trends",
+            "isRequired": true,
+            "value": {
+              "durationMs": 2419200000
+            },
+            "typeSettings": {
+              "selectableValues": [
+                {
+                  "durationMs": 259200000
+                },
+                {
+                  "durationMs": 604800000
+                },
+                {
+                  "durationMs": 1209600000
+                },
+                {
+                  "durationMs": 2419200000
+                },
+                {
+                  "durationMs": 2592000000
+                },
+                {
+                  "durationMs": 5184000000
+                },
+                {
+                  "durationMs": 7776000000
+                }
+              ],
+              "allowCustom": true
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "label": "Time Range"
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "Parameters"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "## Number Of Devices Trends\r\n---"
+      },
+      "name": "text - NumberOfDevicesTrends"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "IntuneDevices\r\n| where Result != \"Completed\"\r\n| summarize NumberOfDevices = dcount(DeviceId) by BatchId\r\n| join (\r\nIntuneDevices\r\n| where Result == \"Completed\"\r\n| project BatchTime = TimeGenerated, BatchId\r\n| summarize arg_max(BatchId, *) by BatchId\r\n) on BatchId\r\n| project BatchTime, NumberOfDevices",
+        "size": 3,
+        "aggregation": 5,
+        "showAnnotations": true,
+        "timeContext": {
+          "durationMs": 0
+        },
+        "timeContextFromParameter": "TimeRange",
```

Comment:
this query seems to be the only one using the time range parameter?

Reply:
Yes, we only use Time Range for trends query. The other queries will based on the latest daily snapshot (> ago(1day).

Reply:
it's kindof a strange thing though, that one shows me what i picked, and the others don't.  i'm not the PM of your feature but it is a little strange.